### PR TITLE
update monitor.

### PR DESCRIPTION
**Pull request by Alice1319**
fixes eclipse-xpanse/xpanse#437.

image

### CODE REVIEW

File: src/components/content/monitor/MonitorChart.tsx [hunk 1]
```diff
@@ -3,50 +3,326 @@
...
+                    setOnlyLastKnownMonitorMetricsQueue([]);
+                    stopFetchOnlyLastKnownMonitorMetricDataTimer();
+                    getTipInfo('', false, 'success', 'No metrics found for the selected service.', true);
+                }
+            })
+            .catch(() => {
+                setIsAutoRefresh(false);
+                setOnlyLastKnownMonitorMetricsQueue([]);
+                stopFetchOnlyLastKnownMonitorMetricDataTimer();
+                getTipInfo('', false, 'error', 'Error while fetching metrics data.', true);
```
Comment (swaroopar) [inline src/components/content/monitor/MonitorChart.tsx:205]: for all the catch blocks which we have for API calls, we must extract the error out of it and display it in the alert description. 

Example - https://github.com/eclipse-xpanse/xpanse-ui/blob/f845b6938c95e8c84ad2a410ea9d6592a634e59a/src/components/content/register/registerService.ts#L27

Comment (Alice1319) [reply src/components/content/monitor/MonitorChart.tsx]: fixed.

File: src/components/content/monitor/metricProps.ts [hunk 2]
```diff
@@ -15,20 +15,38 @@ export interface MetricProps {
...
+    activeMonitorMetricType: string;
+
+    onlyLastKnownResult: boolean;
+    timeInterval: number;
+    timePeriod: number;
+}
+
+export const chartsPerRowCountList: string[] = ['2', '3', '4', '6', '8', '12'];
+export const chartsPerRowWithTwo: string = '2';
+export const timePeriodInLastMinute: number = 1;
```
[the remark this answers is not in the document]
Comment (Alice1319) [reply src/components/content/monitor/metricProps.ts]: fixed.

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**